src/provider.rs

```diff
@@ -99,6 +114,30 @@ pub struct Imgur;
 impl Provider for Imgur {
     type Lifetime = Refresh;
     type Token = Bearer<Refresh>;
-    fn auth_uri(&self) -> &str { "https://api.imgur.com/oauth2/authorize" }
-    fn token_uri(&self) -> &str { "https://api.imgur.com/oauth2/token" }
+    fn auth_uri(&self) -> &Url { &IMGUR_AUTH_URI }
+    fn token_uri(&self) -> &Url { &IMGUR_TOKEN_URI }
 }
+
+#[test]
+fn google_urls() {
+    let prov = google::Web {};
```

Comment:
Braces like this are valid?

Reply:
Its empty struct initialization. Its the same syntax as initializing a struct with all pub fields like `let foo = Bar { baz: "hello" }` just without any member data.

Reply:
But why? The structs are unit structs.

Reply:
I was probably thinking something like "remind anyone reading the test you can have fields in your Providers" but they are redundant so I got rid of them!